cfgov/cfgov/settings/base.py

```diff
@@ -479,18 +479,26 @@
 REGSGOV_BASE_URL = os.environ.get('REGSGOV_BASE_URL')
 REGSGOV_API_KEY = os.environ.get('REGSGOV_API_KEY')
 
-# Akamai
+# CDNs
+WAGTAILFRONTENDCACHE = {}
+
 ENABLE_AKAMAI_CACHE_PURGE = os.environ.get('ENABLE_AKAMAI_CACHE_PURGE', False)
 if ENABLE_AKAMAI_CACHE_PURGE:
-    WAGTAILFRONTENDCACHE = {
-        'akamai': {
-            'BACKEND': 'v1.models.akamai_backend.AkamaiBackend',
-            'CLIENT_TOKEN': os.environ.get('AKAMAI_CLIENT_TOKEN'),
-            'CLIENT_SECRET': os.environ.get('AKAMAI_CLIENT_SECRET'),
-            'ACCESS_TOKEN': os.environ.get('AKAMAI_ACCESS_TOKEN')
-        },
+    WAGTAILFRONTENDCACHE['akamai'] = {
+        'BACKEND': 'v1.models.caching.AkamaiBackend',
+        'CLIENT_TOKEN': os.environ.get('AKAMAI_CLIENT_TOKEN'),
+        'CLIENT_SECRET': os.environ.get('AKAMAI_CLIENT_SECRET'),
+        'ACCESS_TOKEN': os.environ.get('AKAMAI_ACCESS_TOKEN')
     }
 
+ENABLE_CLOUDFRONT_CACHE_PURGE = os.environ.get('ENABLE_CLOUDFRONT_CACHE_PURGE', False)
```

Comment:
If both Akamai and Cloudfront are enabled, how does Wagtail know which backend to use when publishing a page?

Reply:
Nevermind just saw the logic in `admin_views.py`!